src/content/topics/home/using-flags/flag-settings.mdx

```diff
@@ -46,6 +46,12 @@ Here is an image of the settings for all environments on the flag's **Settings**
 
 The flag maintainer is the account member who is primarily responsible for the flag. By default, the maintainer is set to the member who created the flag, but you can assign any member of your team as the maintainer for a particular flag.
 
+Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that if one member leaves the team, the flag still has other maintainers.
+
+The flag's maintainer can be updated on the flag's **Settings** tab, or from the header of any of the flag's tabs:
+
+![Clicking the pencil icon in the header of the flag page allows you to set a new flag maintainer.](edit-flag-maintainer.png)
```

Comment:
```suggestion
You can update the flag's maintainer on the flag's **Settings** tab. You can also update the maintainer by clicking the pencil icon from the header of any of the flag's tabs:

![The flag's maintainer and "pencil" icon.](edit-flag-maintainer.png)
```

Style nits -- active voice, and our captions [describe what the reader sees](https://launchdarkly.atlassian.net/wiki/spaces/PDH/pages/342786398/Style+guide+Tech+Writing+Documentation#Screenshot-captions), rather than the action they might do from here.

Reply:
Ah, thank you!